### PR TITLE
Update security.rst

### PR DESCRIPTION
**Pull request by mpaquet**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.7+ |
| Fixed tickets | #5698 |

### CODE REVIEW

File: book/security.rst [hunk 1]
```diff
@@ -510,11 +510,16 @@ else, you'll want to encode their passwords. The best algorithm to use is
             // ...
         ));
 
-.. include:: /cookbook/security/_ircmaxwell_password-compat.rst.inc
+.. include:: /cookbook/security/_ircmaxwell_password-compat.rst.inco
```
Comment (WouterJ) [inline book/security.rst]: you made a little typo here. Can you please fix that?

Comment (mpaquet) [inline book/security.rst]: Indeed, sorry !
This is fixed.

**Comment by WouterJ** [comment]
Thank you for your quick response & great PR! You've my :+1:  :)

**Comment by xabbuh** [comment]
Thank you @mpaquet for this great pull request. I have squashed your commits while merging. That's why your PR is shown as closed. I then change `bin/console` to `app/console` on the `2.7` branch in 4ee6cb2ed147eacb95cb9b147e6144c6018ab104 and finally reverted that change for Symfony 3.0 in 7d922b9058f1364cb36e3e736f5f1418ad7397a4.

**Pull request closed without merging**